lib/linux_admin/hosts.rb

```diff
@@ -26,6 +26,10 @@ def add_alias(address, hostname, comment = nil)
 
     alias_method :update_entry, :add_alias
 
+    def set_loopback_hostname(hostname, comment = nil)
+      ["::1", "127.0.0.1"].each { |address| add_name(address, hostname, true, comment, false) }
```

Comment:
@bdunne That had crossed my mind too. It turns out duplicates are removed already here:
https://github.com/jvlcek/linux_admin/blob/09f99dabc5578676432d609b6a069e3dbd685d70/lib/linux_admin/hosts.rb#L101